### DISCUSSION TITLE
Help! My email address can't receive OTP email anymore 😢

### DISCUSSION THREAD

**Discussion by Zimmerit** [npm/feedback#942] [General]
Hello!

This email address `developer@metamaker.cn` couldn't receive any email from npmjs.org. 
For comparsion I tested this address can receive email from other email address, and other email address can receive emails from npmjs.org. The email system works fine, so I think I can only get help from npmjs.org. I wonder whether my address was blocked or not , and please let me know the reason I will do my effort to avoid it.

Thanks for your patience!

**Comment by monishcm** [comment]
Please reach out to our support @ https://www.npmjs.com/support
